Resources/Prototypes/Species/human.yml

```
@@ -40,27 +47,210 @@
+  id: HumanNormal
+  name: body-normal
+  sprites:
+    Special: MobHumanoidAnyMarking
+    Head: MobHumanHeadOld
+    Hair: MobHumanoidAnyMarking
+    FacialHair: MobHumanoidAnyMarking
+    Snout: MobHumanoidAnyMarking
+    Chest: MobHumanTorsoOld
+    Eyes: MobHumanoidEyes
+    HeadTop: MobHumanoidAnyMarking
+    Tail: MobHumanoidAnyMarking #Sunrise-edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)